Added the SodaSeti theme to the repository

domibarton opened the pull request.

FichteFoll commented:
You'll need to change `"branch": "master"` to `"tags": true` and use [tags](http://git-scm.com/book/en/v2/Git-Basics-Tagging) to create releases and push valid [**semantic version**](http://semver.org) tags to your repo. As mentioned [in the docs](https://packagecontrol.io/docs/submitting_a_package#Step_4), we no longer accepts branch-based releases for new packages.

You can not use submodules to install packages via Package Control. They are not supported.

domibarton commented:
You mean "RTFM" - sorry my fault ;)

I removed the git submodule, created a new v1.0.0 tag in my repo and updated your repo file to use tags instead of the master branch.

Hope that's OK?
